main/src/observables/gravitational_waves.hpp

```diff
@@ -77,69 +77,157 @@ std::array<T, 8> gravRad(Dataset& d,size_t first, size_t last, T viewTheta, T vi
+                                             d.ay.data(),
+                                             d.az.data(),
+                                             d.m.data());
+
+    int              rootRank = 0;
     std::array<T, 6> d2Qxx_global;
-    MPI_Reduce(d2Q_local.data(), d2Qxx_global.data(),6, MpiType<T>{}, MPI_SUM, rootRank, MPI_COMM_WORLD);
+    MPI_Reduce(d2Q_local.data(), d2Qxx_global.data(), 6, MpiType<T>{}, MPI_SUM, rootRank, MPI_COMM_WORLD);
 
     T httplus;
     T httcross;
 
     computeHtt(d2Qxx_global, viewTheta, viewPhi, &httplus, &httcross);
 
-    return {httplus, httcross, d2Qxx_global[0], d2Qxx_global[1], d2Qxx_global[2], d2Qxx_global[3], d2Qxx_global[4], d2Qxx_global[5]};
+    return {httplus,
+            httcross,
+            d2Qxx_global[0],
+            d2Qxx_global[1],
+            d2Qxx_global[2],
+            d2Qxx_global[3],
+            d2Qxx_global[4],
+            d2Qxx_global[5]};
 }
 
-
-//! @brief Observables that includes times, energies, gravitational radiation and the second derivative of the quadrupole moment
+/*! @brief Observables that includes times, energies, gravitational radiation and the second derivative of the
+ * quadrupole moment
+ */
 template<class Dataset>
 class GravWaves : public IObservables<Dataset>
 {
     using T = typename Dataset::RealType;
     std::ofstream& constantsFile;
-    T viewTheta;
-    T viewPhi;
+    T              viewTheta;
+    T              viewPhi;
 
 public:
     GravWaves(std::ofstream& constPath, T theta, T phi)
-        : constantsFile(constPath), viewTheta(theta), viewPhi(phi)
+        : constantsFile(constPath)
+        , viewTheta(theta)
+        , viewPhi(phi)
     {
     }
 
-
-
     void computeAndWrite(Dataset& d, size_t firstIndex, size_t lastIndex, cstone::Box<T>& box)
     {
-
         std::array<T, 8> out = gravRad(d, firstIndex, lastIndex, viewTheta, viewPhi);
 
         int rank;
         MPI_Comm_rank(d.comm, &rank);
 
         if (rank == 0)
         {
-            fileutils::writeColumns(
-                constantsFile, ' ', d.iteration, d.ttot, d.minDt, d.etot, d.ecin, d.eint, d.egrav,
-                out[0], out[1], out[2], out[3], out[4], out[5], out[6], out[7]);
+            fileutils::writeColumns(constantsFile,
+                                    ' ',
+                                    d.iteration,
+                                    d.ttot,
+                                    d.minDt,
+                                    d.etot,
+                                    d.ecin,
+                                    d.eint,
+                                    d.egrav,
+                                    out[0],
+                                    out[1],
+                                    out[2],
+                                    out[3],
+                                    out[4],
+                                    out[5],
+                                    out[6],
+                                    out[7]);
```

Comment:
Same here. Out is a meaningless variable. It should have a name that has meaning to understand what is returning and where.

Reply:
same